doc/tools/make_rst.py

```diff
@@ -1443,6 +1487,28 @@ def make_getter_signature(class_def: ClassDef, property_def: PropertyDef, state:
     return f"{ret_type} {signature}"
 
 
+def make_deprecated_experimental(item: DefinitionBase, state: State) -> str:
+    result = ""
+
+    if item.deprecated is not None:
+        deprecated_prefix = translate("Deprecated:")
+        if item.deprecated.strip() == "":
+            default_message = translate(f"This {item.definition_name} may be changed or removed in future versions.")
+            result += f"**{deprecated_prefix}** {default_message}\n\n"
+        else:
+            result += f"**{deprecated_prefix}** {format_text_block(item.deprecated.strip(), item, state)}\n\n"
```

Comment:
Is there a way to use color in RST? This will probably require some changes to the site so that we can use:

```rst
:red:`Deprecated:"` Message.
```

or

```rst
.. deprecated:: Message.
```

Now it looks like this:

![](https://github.com/godotengine/godot/assets/47700418/c053a54b-6f59-48e3-8e7b-41a7eb6bb532)

I only added this to the descriptions, not to the overviews.

Reply:
RST doesn't support arbitrary colors (aside of raw HTML tags).

Reply:
[How to use color in text with ReStructured Text (rst2html.py) or how to insert HTML tags without blank lines?](https://stackoverflow.com/questions/4669689/how-to-use-color-in-text-with-restructured-text-rst2html-py-or-how-to-insert-h)